### ISSUE TITLE
"Chaos per map" sometimes is wrong 

### ISSUE THREAD

**Issue by Memoframe** [deathbeam/maps-of-exile#25]
"Chaos per map" sometimes is wrong.

1. Search for "The Chains that Bind"
2. Hover over card name on each map entry that drops it.
3. For some maps it's 1 card per 1 map; for some maps it's 1 card per 2 map. But in both cases profit per map is the same.

Expected behavior:
Correct profit per map.

Desktop:
 - Linux
 - Chrome
 - 118.0.5993.117 (Official Build) (64-bit)

Additional context
Just thoughts: maybe some weird rounding issue?

**Comment by deathbeam** [comment]
Yea could be rounding error as its a lot of floating math with very small numbers. As the site first calculates amount of drop pool items you get for your choicec and then multiples weigtht/total drop pool size with that, and total drop pool is very big so the resulting number is small.

And then I did not wanted to show partial numbers when showing cards per map so I always round up so even if the number is off by very very small floating point it gets rounded to 2.

I just went through all the cards and I did not saw it get rounded to 2 so its also probably browser or even CPU dependant. I can maybe show partial numbers instead of always rounding up and that could maybe solve it at least partially so it do not looks so off when this happens, not sure.
